app/src/main/res/values-fr/strings.xml

```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
     <string name="pref_name_misc_wipeDB">Purger la base de données</string>
     <string name="pref_name_misc_wipeDB_confirmTitle">Purger la base de données ?</string>
     <string name="pref_name_misc_wipeDB_confirmMessage">Êtes vous certain de vouloir purger la base de données ?</string>
+    <string name="checkFeeds_progressDialogMessage">Vérification des flux</string>
+    <string name="connectionWizard_providerSelection_next">Suivant</string>
+    <string name="connectionWizard_providerSelection_prev">Ignorer</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_desc">Choisissez cette option si vous avez un compte sur https://framabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_name">Framabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_general_desc">Choisissez cette option pour entrer les paramètres requis manuellement</string>
+    <string name="connectionWizard_providerSelection_provider_general_name">Autre</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_desc">Choisissez cette option si vous avez un compte sur https://v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_name">v2.wallabag.org</string>
```

Comment:
when my PR is merged, this line is going to be unnecessary
https://github.com/wallabag/android-app/pull/297/files#diff-01dafb3fd0217441330103cfc8300094L221